inst/app_template/tests/testthat/test-sort.R

```diff
@@ -1,6 +1,4 @@
 # Test the lexical_sort function from R/example.R
```

Comment:
We also recommend naming the test file to match the R file so this should be `test-example.R`